### PR TITLE
feat(wedlocks) - add custom email content for keyMined template

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: wedlocks/src/templates/keyMined.js [hunk 1]
```diff
@@ -9,6 +9,8 @@ export default {
 
 <p>A new membership (#{{keyId}}) to the lock <strong>{{lockName}}</strong> was just minted for you!</p>
 
+{{customEmailContent}}
```
Comment (kalidiagne) [inline wedlocks/src/templates/keyMined.js:12]: @julien51 this will cause extra spacing when the custom content is not present? 🤔

Comment (searchableguy) [reply wedlocks/src/templates/keyMined.js:12]: Use conditional 

https://handlebarsjs.com/examples/builtin-helper-ifelse-block.html

**Review by julien51 — COMMENTED** [review]
Approach looks good to me. Please add extensive tests here.

**Comment by kalidiagne** [comment]
templates will be in `email-templates` packages, ill move everything in that packages as soon we merge this PR https://github.com/unlock-protocol/unlock/pull/10980

**Pull request closed without merging**